docs/guides/src/main/server/configuration.adoc

```diff
@@ -145,38 +139,35 @@ You can also specify a different configuration file by using the `[-cf|--config-
+=== Development versus production mode
 
-* Development
-* Production
+The server supports the following operating modes:
 
-The development mode is activated every time you run the `start-dev` command. In this mode, some key configuration options are set to make it possible to start the
+Development mode:: This mode is activated every time you run the `start-dev` command. In this mode, some key configuration options are set to make it possible to start the
 server for development purposes without the burden of having to define additional settings that are mandatory for production.
 
-The production mode is activated by default when you run the `build` or the `start` command. Use this mode to set any configuration option that
+Production mode:: This mode is activated by default when you run the `build` or the `start` command. Use this mode to set any configuration option that
 is needed for deploying Keycloak in production.
 
-By default, the configurations options for the production mode are commented out in the `conf/keycloak.conf`. These examples
- are meant to give you an idea about the main settings that needs to be considered when running in production.
+By default, the configuration options for the production mode are commented out in the `conf/keycloak.conf`. These examples
+ are meant to give you an idea about the main settings to consider when running in production.
 
-== Using unsupported server options
+== Unsupported server options
 
-Most of the time the available options from the server configuration should be enough to configure the server.
-However, you might need to use properties directly from Quarkus in order to enable a specific behavior or capability that is missing from the server configuration.
+In most cases, the available options from the server configuration should suffice to configure the server.
+However, you might need to use properties directly from Quarkus to enable a specific behavior or capability that is missing from the server configuration.
 
-You should avoid as much as possible using properties directly from Quarkus. If you really need to, consider opening an https://github.com/keycloak/keycloak/issues/new?assignees=&labels=kind%2Fenhancement%2Cstatus%2Ftriage&template=enhancement.yml[issue] first and help us
+As much as possible, avoid using properties directly from Quarkus. If your need is essential, consider opening an https://github.com/keycloak/keycloak/issues/new?assignees=&labels=kind%2Fenhancement%2Cstatus%2Ftriage&template=enhancement.yml[issue] first and help us
 to improve the server configuration.
 
-To configure the server using Quarkus properties you should follow these steps:
+To configure the server using Quarkus properties, perform the following steps:
 
-* Create a `conf/quarkus.properties` file and define any property you need
-* Run the `build` command to apply the settings to the server
+. Create a `conf/quarkus.properties` file and define any property you need.
```

Comment:
Not 100% sure, but shouldn't these stay bullet points of an unordered list? (a `*`)

Reply:
you are right, was a bit confused, sorry. so when you are changing the whitespace, feel free to also change back to step / dots.